tests/test_indexer.py

```diff
@@ -71,6 +72,7 @@ class ESInfo:
     server = None
 
 def setUpModule():
+    IndexSuffix.name = __name__.split('.')[-1]
```

Comment:
you could consider `rsplit(".", 2)` or `rpartition(".")` or you could leave it as is.

Reply:
i'll use rsplit, it'll be a little faster